src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -150,8 +150,7 @@ const MemoisedInner = ({
     if (Banner) {
         return (
             // The css here is necessary to put the container div in view, so that we can track the view
-            <div ref={setNode} className={emotion.css`position: fixed; bottom: -1px;`}>
-                hello
+            <div ref={setNode} className={emotion.css`position: fixed; bottom: -1px; width: 100%; z-index: 999`}>
```

Comment:
We'll probably lint use of z-index, you want https://github.com/guardian/dotcom-rendering/blob/master/src/web/lib/getZIndex.tsx

Reply:
👍